scan/lib/scan/detect_values.rb

```diff
@@ -24,9 +24,22 @@ def self.set_additional_default_values
       default_device_tvos if Scan.project.tvos?
       detect_destination
 
+      default_derived_data
+
       return config
     end
 
+    def self.default_derived_data
+      return unless Scan.config[:derived_data_path].to_s.empty?
+      default_path = Scan.project.build_settings(key: "BUILT_PRODUCTS_DIR")
+      # => /Users/.../Library/Developer/Xcode/DerivedData/app-bqrfaojicpsqnoglloisfftjhksc/Build/Products/Release-iphoneos
+      # We need to go one folder up, since tests don't run inside 'Release-iphoneos'
+      # but in 'Debug-iphonesimulator'
+      default_path = File.join(File.expand_path("..", default_path), "Debug-iphonesimulator")
```

Comment:
I know some people in the past have tried to make changes to allow `scan` to run on real devices. Does this `Debug-iphonesimulator` path assume too much?